### ISSUE TITLE
Disabled miners still get banchmarked

### ISSUE THREAD

**Issue by smyers63** [RainbowMiner/RainbowMiner#2216]
I purposely disabled all the instances of BzMiner and MultiMiner in the Miners section of the localhost interface before running benchmarks again.  Those 2 miners fail for every pool/coin/OC profile so I didn't want time wasted benchmarking those again.  They were benchmarked anyway.

Is this normal behavior?  How can I prevent failing miners from being benchmarked over and over?

**Comment by RainbowMiner** [comment]
No, this is a bug - until I fix this, I recommend you put both miners on the ExcludeMinerName list in config.txt (or use the localhost:4000/setup.html Pools, Miners and Algorithms section). That should really shut them off.

**Comment by smyers63** [comment]
That was going to be my next step.  I'll do that.  Thanks.
